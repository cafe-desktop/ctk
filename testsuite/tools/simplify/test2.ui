<!-- non-default property not stripped -->
<interface>
  <object class="CtkWindow">
    <property name="visible">True</property>
  </object>
</interface>
