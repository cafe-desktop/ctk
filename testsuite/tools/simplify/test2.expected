<!-- non-default property not stripped -->
<interface>
  <object class="GtkWindow">
    <property name="visible">1</property>
  </object>
</interface>
